test/valgrind/valgrindTestSuite.sh

```diff
@@ -352,6 +353,21 @@ function processResult()
 
 # -----------------------------------------------------------------------------
 #
+# Extract error info from valgrind output file
+#
+# It uses as argument the .out file to process
+#
+function memErrorInfo()
+{
+  filename=$1
+  # Get info from valgrind
+  memErrors=$(grep -n "Invalid write of size" $filename | wc -l)
```

Comment:
Maybe could even add a "invalid read" to the check? Not sure if we have some case like that but better to be prepared for that...